Travis fails with unrelated 
```The command "sudo -E apt-add-repository -y "ppa:ubuntu-toolchain-r/test"" failed and exited with 1 during .```

@rryan: Could you have a look?  

And related: 
```
src/track/bpm.h:47:16: error: no member named 'round' in namespace 'std'; did you mean simply 'round'?

        return std::round(value);

               ^~~~~~~~~~

               round
``` 

std::round() is a c++11 function. It looks like just round will fix the issue here. 
On Mac, <math.h> seams not to include <cmath.h> so I think we have to include both headers. 
We may do it conditional for GCC >= 6.1.1 

Any thoughts?

--- a/src/util/math.h
+++ b/src/util/math.h
@@ -11,7 +11,11 @@
 #endif
 
 #include <math.h>
-// Note: #include <cmath> does not work with GCC 6.1.1 because of our fpclassify hack 
+#include <cmath> 
+// Note: Because of our fpclassify hack, we actualy need to inlude both, 
+// the c and the c++ version of the math header.  
+// From GCC 6.1.1 math.h depends on cmath, which failes to compile if included 
+// after our fpclassify hack 
 
 #include <algorithm>
 

